### PR TITLE
Add test-e2e make target

### PR DESCRIPTION
**Pull request by sanchezl**
Layout initial support for adding e2e tests.

### CODE REVIEW

**Comment by deads2k** [comment]
@sanchezl looks like the first task is getting a bump to the current library-go.  I think you'll need to figure out what the pull that added https://github.com/openshift/cluster-kube-apiserver-operator/pull/371/commits/40cd15b88afa3dc751695e87d16dff713282e7ab broke.  KASo may be the easiest spot for that.

File: glide.yaml [hunk 1]
```diff
@@ -28,3 +28,7 @@ import:
   version: a0ff2567cfb70903282db057e799fd826784d41d
 - package: k8s.io/gengo/args
   version: 01a732e01d00cb9a81bb0ca050d3e6d2b947927b
+- package: github.com/stretchr/testify
+  version: ~1.3.0
```
Comment (deads2k) [inline glide.yaml]: no.  tag or SHA

File: test/e2e/operator_test.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	test "github.com/openshift/cluster-kube-controller-manager-operator/test/library"
+)
+
+func TestOperatorNamespace(t *testing.T) {
+	kubeConfig, err := test.NewClientConfigForTest()
+	require.NoError(t, err)
```
Comment (deads2k) [inline test/e2e/operator_test.go]: such a useless dep.  Who wanted this?  I won't straight out nak, but I haven't seen any real utility.

**Comment by deads2k** [comment]
fix the version/SHA and I can merge.

File: test/e2e/operator_test.go (hunk 2 above)
Comment (sanchezl) [reply test/e2e/operator_test.go]: I removed it.

**Pull request merged**